src/__tests__/pages/policy/input/ParameterEditor.test.js

```diff
@@ -0,0 +1,52 @@
+// CenteredMiddleColumn.test.js
```

Comment:
Just to be clear, do you plan to separate this out into a new file later? If not, it would be best if you could do so and mirror the folder structure of the repo as a whole.